quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -51,6 +51,7 @@
 SECRETS_DIR      = 'secrets'
 SA_KEY_PATH      = SECRETS_DIR + '/gcp_service_account_key.json'
```

Comment:
probably want to call this GCP_SA_KEY_PATH now that there are multiple SAs involved.